seerpy/seerpy.py

```diff
@@ -1188,14 +1188,24 @@ def get_diary_labels(self, patient_id, label_type='all', offset=0, limit=100, fr
                     ]):
                         query_flag = True
                     break
-
                 label_results['labelGroups'][idx]['labels'].extend(labels)
 
             offset += limit
 
+        # filter
+        for idx, group in enumerate(label_results['labelGroups']):
+            filtered_labels = []
+            labels = group['labels']
+            if labels:
+                for label in labels:
+                    if any(tag['tagType']['value'].lower() == tag_type
+                        for tag in label['tags']):
+                        filtered_labels.append(label)
```

Comment:
```suggestion
                            filtered_labels.append(label)
```

Missing indentation?